web/src/components/tasks/CannabisPriorityStatus.tsx

```diff
@@ -0,0 +1,342 @@
+        else return false;
+      });
+
+      const socialEquityPriorityTypeSelected = priorityTypesObj.socialEquity.some((key) => {
+        if (key in userData.taskItemChecklist) return userData.taskItemChecklist[key] === true;
+        else return false;
+      });
+
+      setDisplayMWPriorityType(minorityOrWomenPriorityTypeSelected ? true : false);
+      setDisplayVeteranPriorityType(veteranPriorityTypeSelected ? true : false);
+      setDisplayImpactZonePriorityType(impactZonePriorityTypeSelected ? true : false);
+      setDisplaySocialEquityPriorityType(socialEquityPriorityTypeSelected ? true : false);
+      setDisplayNoPriorityType(userData.taskItemChecklist[priorityTypesObj.noneOfTheAbove] ? true : false);
+    }
+  }, [userData, update, priorityTypes, displayFirstTab]);
+
+  const handleCheckboxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.checked) {
+      const unselectPriorityTasks = {} as Record<string, boolean>;
+      priorityTypes.forEach((key) => (unselectPriorityTasks[key] = false));
+
+      update({
+        ...userData,
+        taskItemChecklist: {
+          ...userData.taskItemChecklist,
+          ...unselectPriorityTasks,
+          [priorityTypesObj.noneOfTheAbove]: true,
+        },
+      });
+    } else {
+      update({
+        ...userData,
+        taskItemChecklist: {
+          ...userData.taskItemChecklist,
+          [priorityTypesObj.noneOfTheAbove]: false,
+        },
+      });
+    }
+  };
+
+  const handleNextTabButtonClick = () => {
+    setDisplayFirstTab(false);
+    scrollToTop();
+    if (
+      userData.taskProgress[props.task.id] === undefined ||
+      userData.taskProgress[props.task.id] === "NOT_STARTED"
+    ) {
+      setSuccessToastIsOpen(true);
+      update({
+        ...userData,
+        taskProgress: {
+          ...userData.taskProgress,
+          [props.task.id]: "IN_PROGRESS",
+        },
+      });
+    }
+  };
+
+  const handleBackButtonClick = () => {
+    setDisplayFirstTab(true);
+    scrollToTop();
+  };
+
+  const handleCompleteTaskButtonClick = () => {
+    setSuccessToastIsOpen(true);
+    update({
+      ...userData,
+      taskProgress: {
+        ...userData.taskProgress,
+        [props.task.id]: "COMPLETED",
+      },
+    });
+  };
+
+  const renderFirstTabContent = (
+    <>
+      <Content>{props.task.contentMd}</Content>
+      <div className="usa-prose">
+        <ul>
+          <div className="margin-y-2">
+            <FormControlLabel
+              label={Config.cannabisPriorityStatus.noPriortyStatusCheckboxText}
+              control={
+                <Checkbox
+                  onChange={handleCheckboxChange}
+                  checked={
+                    userData && userData.taskItemChecklist[priorityTypesObj.noneOfTheAbove]
+                      ? userData.taskItemChecklist[priorityTypesObj.noneOfTheAbove]
+                      : false
```

Comment:
similar comment as above, i think this could be simplified to `userData?.taskItemChecklist[priorityTypesObj.noneOfTheAbove] === true`